cmd/aws-vpc-cni/main.go

```diff
@@ -384,14 +387,17 @@ func validateEnvVars() bool {
 	}
 
 	prefixDelegationEn := utils.GetBoolAsStringEnvVar(envEnPrefixDelegation, defaultEnPrefixDelegation)
-	warmIPTarget := utils.GetEnv(envWarmIPTarget, "0")
-	warmPrefixTarget := utils.GetEnv(envWarmPrefixTarget, "0")
-	minimumIPTarget := utils.GetEnv(envMinIPTarget, "0")
+	warmIPTarget := utils.GetEnv(envWarmIPTarget, defaultWarmIPTarget)
+	warmPrefixTarget := utils.GetEnv(envWarmPrefixTarget, defaultWarmPrefixTarget)
+	minimumIPTarget := utils.GetEnv(envMinIPTarget, defaultMinIPTarget)
 
 	// Note that these string values should probably be cast to integers, but the comparison for values greater than 0 works either way
 	if prefixDelegationEn && (warmIPTarget <= "0" && warmPrefixTarget <= "0" && minimumIPTarget <= "0") {
 		log.Errorf("Setting WARM_PREFIX_TARGET = 0 is not supported while WARM_IP_TARGET/MINIMUM_IP_TARGET is not set. Please configure either one of the WARM_{PREFIX/IP}_TARGET or MINIMUM_IP_TARGET env variables")
 		return false
+	} else if warmIPTarget <= "0" && minimumIPTarget > "0" {
```

Comment:
nit: with the int values above, we can change this from string checks to integer checks. Also, we should allow `WARM_IP_TARGET=0` when `MINIMUM_IP_TARGET` is set. We just want to default `WARM_IP_TARGET` to 1 in this case (env var is unset)